keylime/cli/policies.py

```diff
@@ -43,6 +46,13 @@ def process_allowlist(args):
         # Auto-enable IMA (or-bit mask)
         tpm_policy["mask"] = hex(int(tpm_policy["mask"], 0) | (1 << config.IMA_PCR))
 
+        logger.warning(
+            """
+            WARNING: Verification key support in the Keylime tenant is deprecated. Provide
+            verification keys as part of an IMA policy instead.
+            """
+        )
```

Comment:
```suggestion
        logger.warning(
            "WARNING: Verification key support in the Keylime tenant is deprecated. Provide "
            "verification keys as part of an IMA policy instead."
        )
```

The `"""` will include the spaces.

Reply:
Fixed, thanks!